src/it/scala/net/snowflake/spark/snowflake/SnowflakeResultSetRDDSuite.scala

```diff
@@ -419,7 +419,7 @@ class SnowflakeResultSetRDDSuite extends IntegrationSuiteBase {
                   | int_c int, c_string string(1024) )""".stripMargin)
 
     jdbcUpdate(s"""insert into $test_table_large_result select
-                  | seq4(), '$largeStringValue'
+                  | row_number() over (order by seq4()) - 1, '$largeStringValue'
```

Comment:
The result generated by `seq4()` may not be continues. This may cause some test case failed.